### ISSUE TITLE
Incorrect documentation or missing code

### ISSUE THREAD

**Issue by mr-stringer** [bjin01/salt-sap-patching#1]
The README.md has a text block showing the output of `tree /srv/salt`.

This listing shows a file named `patch_hana.py` in `_modules` but this file in not present in the code.

Is the listing incorrect or is the file missing?

**Comment by bjin01** [comment]
You are right. the file patch_hana.py does not exist in the _modules directory and this file is also not needed. The file patch_hana.py was a test file and existed in the directory _modules shortly. I will remove it in the readme.md.

Thanks for this report.
